src/styles/pages/get-involved/donate.module.scss

```diff
@@ -0,0 +1,51 @@
+.sectionContainer {
+  display: flex;
+  padding-top: 32px;
+  background-color: var(--color-white);
+  font-family: var(--font-inter);
+  padding: var(--edge-spacer);
+}
+
+.donateBox {
+  margin-top: 16px;
+  background-color: var(--color-beige);
+  padding-top: 16px;
+  padding-bottom: 16px;
+  padding-left: 32px;
+  padding-right: 32px;
+  border-radius: 5px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
```

Comment:
You don't really need to define the width here because the left-right padding you added to its parent is taking care of the width.